### PR TITLE
feat(cab-provider) : provider profile screens

### PR DESCRIPTION
**Pull request by naghmakhan13**
Added the Cab-Provider profile first 3 screens.

### CODE REVIEW

File: DVSRegister/Views/CabProvider/BeforeYouStart.cshtml
Comment (AiswaryaBEIS) [inline DVSRegister/Views/CabProvider/BeforeYouStart.cshtml:1]: Back links in all views to be updated

File: DVSRegister/Controllers/CabProviderController.cs [hunk 1]
```diff
@@ -12,7 +12,7 @@
 namespace DVSRegister.Controllers
 {
     [Route("cab-provider")]
-    [ValidCognitoToken]
+    /*[ValidCognitoToken]*/
```
[the remark this answers is not in the document]
Comment (naghmakhan13) [reply DVSRegister/Controllers/CabProviderController.cs:15]: Added

File: DVSRegister/Controllers/CabProviderController.cs [hunk 2]
```diff
@@ -31,6 +31,126 @@ public IActionResult BeforeYouStart()
         {            
             return View();
         }
+        //First 5 Screens
```
[the remark this answers is not in the document]
Comment (naghmakhan13) [reply DVSRegister/Controllers/CabProviderController.cs:34]: Removed the commented code.

File: DVSRegister/Views/CabProvider/RegisteredName.cshtml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                            ViewData["Title"] = "Error: " + ViewData["Title"];
+                            @await Html.PartialAsync("~/Views/PartialViews/_SpecificErrorsView.cshtml",
+                                     new Tuple<string, IEnumerable<Microsoft.AspNetCore.Mvc.ModelBinding.ModelError>>("RegisteredName", errors))
+                        }
+                    }
+
+                    <p class="govuk-body">This registered name will be publicly displayed on the register of digital identity and attribute service providers. This should match the registered name on the certificate.</p>
+
+                    @{
+                        var websiteViewmodel = new TextBoxViewModel
```
[the remark this answers is not in the document]
Comment (naghmakhan13) [reply DVSRegister/Views/CabProvider/RegisteredName.cshtml:36]: Added

File: DVSRegister/Views/CabProvider/BeforeYouStart.cshtml
Comment (naghmakhan13) [reply DVSRegister/Views/CabProvider/BeforeYouStart.cshtml:1]: Added

**Review by AiswaryaBEIS — APPROVED** [review]
LGTM

**Pull request merged**